.circle/Makefile

```diff
@@ -1,9 +1,13 @@
 ROOT_DIR ?= $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
-NUM_CHANGED_FILES := $(shell $(CI_DIR)/utils/git-changes files | wc -w)
-NUM_CHANGED_PY := $(shell $(CI_DIR)/utils/git-changes py | wc -w)
-NUM_CHANGED_YAML := $(shell $(CI_DIR)/utils/git-changes yaml | wc -w)
-NUM_CHANGED_JSON := $(shell $(CI_DIR)/utils/git-changes json | wc -w)
-CHANGED_DIRECTORIES := $(shell $(CI_DIR)/utils/git-changes directories)
+#NUM_CHANGED_PY := $(shell $(CI_DIR)/utils/git-changes py | wc -w)
```

Comment:
If CI_DIR is not populated correctly, that is concerning. Hmm.

Reply:
It might have been populating correctly but possibly just had the wrong value. GHA doesn't allow string concatenation using a + so I improvised to use the format() function. So I'll look it over and make sure it all matches up knowing what the correct directory should be as a fail safe.

Reply:
And just so you know, this is a testing PR and has a million commits a second. When it checks out , I'll create the final-draft PR which will have a handful of commits just so that it comes across clean.

Reply:
Yes. I expected that. But having it in a PR allows for some discussion about the issues you are working on 😄

Reply:
I'm going to consolidate some of your commits and add them to the gha branch of StackStorm-Exchange/ci.